cmd/cloud-api-adaptor/main.go

```diff
@@ -57,13 +57,13 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 		cmd.Parse("aws", os.Args[1:], func(flags *flag.FlagSet) {
 			flags.StringVar(&awscfg.AccessKeyId, "aws-access-key-id", "", "Access Key ID, defaults to `AWS_ACCESS_KEY_ID`")
 			flags.StringVar(&awscfg.SecretKey, "aws-secret-key", "", "Secret Key, defaults to `AWS_SECRET_ACCESS_KEY`")
-			flags.StringVar(&awscfg.Region, "aws-region", "ap-south-1", "Region")
+			flags.StringVar(&awscfg.Region, "aws-region", "", "Region")
 			flags.StringVar(&awscfg.LoginProfile, "aws-profile", "test", "AWS Login Profile")
 			flags.StringVar(&awscfg.LaunchTemplateName, "aws-lt-name", "kata", "AWS Launch Template Name")
 			flags.BoolVar(&awscfg.UseLaunchTemplate, "use-lt", false, "Use EC2 Launch Template for the Pod VMs")
 			flags.StringVar(&awscfg.ImageId, "imageid", "", "Pod VM ami id")
 			flags.StringVar(&awscfg.InstanceType, "instance-type", "t3.small", "Pod VM instance type")
-			flags.StringVar(&awscfg.SecurityGroupId, "securitygroupid", "", "Security Group Id to be used for the Pod VM")
+			flags.Var(&awscfg.SecurityGroupIds, "securitygroupid", "Security Group Id to be used for the Pod VM")
```

Comment:
The short name be changed should also be changed to `securitygroupids`

Reply:
Another aspect is how the multiple values should be passed from the cli

Reply:
Currently cmdline supports only single SG, the reason i added support to multiple SGs when getting it from IDMS is only because i couldn't tell which i should choose.
I couldn't think of a use-case where multiple values are needed for the cmdline but i'll add support it also if it just the sake of consistency